ShoppingList.ConsoleApp/Models/ShoppingItems/Exceptions/NullArgumentShoppingItemException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using System;
+using Xeptions;
+
+namespace ShoppingList.ConsoleApp.Models.ShoppingItems.Exceptions
+{
+    public class NullArgumentShoppingItemException : Xeption
+    {
+        public NullArgumentShoppingItemException(Exception innerException)
+            : base(message: "Null argument shopping item error occured, fix the errors and try again.",
```

Comment:
occurred typo, check in other instances too

Reply:
Comment Resolved